src/Microsoft.Azure.WebJobs.Extensions.Kafka/Output/KafkaAttribute.cs

```diff
@@ -54,33 +62,33 @@ public KafkaAttribute()
         /// <summary>
         /// Gets or sets the Maximum transmit message size. Default: 1MB
         /// </summary>
-        public int? MaxMessageBytes { get; set; }
+        public int MaxMessageBytes { get => maxMessageBytes.GetValueOrDefault(1000000); set => maxMessageBytes = value; }
```

Comment:
nit: recommend using thousand separators to make it easier to read:

```suggestion
        public int MaxMessageBytes { get => maxMessageBytes.GetValueOrDefault(1_000_000); set => maxMessageBytes = value; }
```

There are other occurrences.